src/platform/polyfills.web.ts

```diff
@@ -9,12 +9,8 @@ window.setImmediate = (cb: () => void) => setTimeout(cb, 0)
 
 // @ts-ignore not on the TS signature due to bad support -prf
 if (!globalThis.Intl?.Segmenter) {
-  // NOTE loading as a separate script to reduce main bundle size, as this is only needed in FF -prf
-  const script = document.createElement('script')
-  script.setAttribute('src', '/static/js/intl-segmenter-polyfill.min.js')
-  document.head.appendChild(script)
-
   // loading emoji mart data
+  // TODO: This condition doesn't make sense; emojimart has nothing to do with Intl.
```

Comment:
Accidentally noticed this. I presume it was added without looking at the condition above. What's happening here is bizarre:

- Emojimart data is undefined in local development (because the condition never triggers) but the emoji picker somehow works. (Is data not required??)
- In production, emojimart data is defined, but it's only because our `scripts.html` partial template adds a `<script>` tag for the thing that triggers emojimart data loading anyway. So this code isn't necessary.

I'd remove it now but I want to understand why it works without data locally. So for now, not touching this.